sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -276,12 +276,18 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
         if (!this.isOpen()) {
           await Promise.race([
             this._init(aborter.signal),
-            delay(retryTimeoutInMs, { abortSignal: aborter.signal }).then(() => {
-              throw {
-                name: "OperationTimeoutError",
-                message: "The management request timed out. Please try again later.",
-              };
-            }),
+            delay(retryTimeoutInMs, { abortSignal: aborter.signal })
+              .then(() => {
+                throw {
+                  name: "OperationTimeoutError",
+                  message: "The management request timed out. Please try again later.",
+                };
+              })
+              .catch((_) => {
+                managementClientLogger.verbose(
+                  `${this.logPrefix} delay of waiting for initialization aborted but it doesn't matter.`,
```

Comment:
Wouldn't this catch both the `AbortError` and the `OperationTimeoutError`?

Reply:
You are right. This should be put before `.then`

Reply:
moved to `.then()`'s onRejected

Reply:
Thanks!

nitpick: could we name the callback to make the code readable?

Also, could we clarify the message a bit? e.g. Initialization experienced a delay and has been aborted.

Reply:
Updated